src/Runner/PhpInterpreter.php

```diff
@@ -24,7 +29,7 @@ function getVersion();
 	/**
 	 * @return bool
 	 */
-	function hasXdebug();
+	function canCodeCoverage();
```

Comment:
:+1: for `canMeasureCodeCoverage`.

---

Wouldn't it be better to ask the `CodeCoverage\Collector` whether he can do it?